doc/lightning-autocleaninvoice.7.txt

```diff
@@ -0,0 +1,42 @@
+LIGHTNING-AUTOCLEANINVOICE(7)
+=============================
+:doctype: manpage
+
+NAME
+----
+lightning-autocleaninvoice - Set up auto-delete of expired invoice
+
+SYNOPSIS
+--------
+*autocleaninvoice* ['cycle_seconds'] ['expired_by']
```

Comment:
I think 'age_in_seconds' or 'expired_for_seconds' is a better name than expired_by, but it's minor.

Reply:
`age` implies we measure from creation time rather than expiry time as we currently behave now.  `expired_for_seconds` is a bit lengthy to me.  `expired_for` is about as unsatisfactory as `expired_by` to me, and the latter is shorter by one character at least.  Hmmm